programs/server/config.xml

```diff
@@ -60,6 +60,18 @@
           </logger>
         </levels>
         -->
+        <!-- Structured log formatting:
+        
+        You can specify log format (for now, JSON only). In that case, the console log will be printed
+        in JSON format.
+        For example, like this below:
+        {"datetime":"2022.03.28 12:17:54.827035","thread_id":"790151","query_id":"2441139","priority":"Debug","source":"KeeperTCPHandler","message":"Received close event with xid 2147483647 for session id #7"}  
```

Comment:
DateTime should be in unix timestamp or in ISO-8601 with UTC.

Reply:
Agreed. Will change this.

Reply:
change submitted in updated PR.